addons/web_editor/static/src/js/wysiwyg/widgets/media_dialog.js

```diff
@@ -141,16 +141,6 @@ var MediaDialog = WeDialog.extend({
         var args = arguments;
         return Promise.resolve(this.active.save()).then(function (data) {
             self.final_data = data;
-            // In the case of multi images selection we suppose this was not to
-            // replace an old media, so we only retrieve the images and save.
-            if (!self.multiImages) {
-                // TODO this dialog triggers 'save' and 'saved' with different
-                // data on close... should refactor to avoid confusion...
-                self.trigger('saved', {
-                    attachments: self.active.images,
```

Comment:
It was used but `_super` was called too so both `save` and `saved` were triggered in that case. The goal of the commit was to adapt the places that listened to `saved` to listen to `save` instead.

Reply:
Yes, my question was: was not this attachment list used ? Because listening to 'save' won't provide you that list. But I guess that list is always empty or with one element in non multi image anyway... I just wonder why that code existed in the first place ?

Reply:
It seems to be introduced in f296992317e96562c66bd7ad59a5080d6c551ed5 and I can't find a place where it would be used even then.

Reply:
:+1: